bbc/simorgh
# WSTEAM1-1174 - UGC campaign 'closed' screen

Resolves JIRA https://jira.dev.bbc.co.uk/browse/WSTEAM1-1174

Overall changes
======
- Adds `ClosedScreen` component to show messaging when the campaign has a `closed` status
- Adds translations for heading and description text of message
<img width="780" alt="Screenshot 2024-07-15 at 11 38 51" src="https://github.com/user-attachments/assets/3d379d32-b0ad-44fd-a124-1d5ae19e50f8">
<img width="248" alt="Screenshot 2024-07-15 at 11 40 00" src="https://github.com/user-attachments/assets/1590e290-2dd2-48da-9e70-f51be90c0562">


Code changes
======
- Adds `ClosedScreen` component
- Adds tests and Storybook stories

Testing
======
1. Visit 'closed' campaign locally: http://localhost:7081/hausa/send/u143732177?renderer_env=live 
2. Visit 'open' campaign to confirm they still render: http://localhost:7081/mundo/send/u50853489?renderer_env=live

Screen-reader checks
======

- [x] Talkback with Chrome
- [x] VoiceOver Mac
- [x] VoiceOver iOS
- [x] JAWS with Chrome
- [x] JAWS with Edge
- [x] NVDA with Firefox
- [x] NVDA with Chrome

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
